### PR TITLE
sgt back end

### PR DESCRIPTION
**Pull request by Lee-SungWon**
Screen Shot 2020-09-25 at 3 40 31 PM

### CODE REVIEW

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    values($1, $2, $3)
+    returning *;
+  `;
+
+  const params = [req.body.name, req.body.course, req.body.grade];
+
+  db.query(sql, params)
+    .then(result => {
+      const grade = result.rows;
+      res.status(201).json(grade);
```
Comment (thebearingedge) [inline sgt-back-end/index.js]: The client should receive only the object and not an array containing the object.

File: sgt-back-end/index.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    where "gradeId" = $2
+  returning *;
+  `;
+
+  const params = [req.body.grade, id];
+
+  db.query(sql, params)
+    .then(result => {
+      const grade = result.rows;
+      if (!grade) {
```
Comment (thebearingedge) [inline sgt-back-end/index.js]: `result.rows` is an array and arrays are always truthy, the array might be empty though (0 length).

File: sgt-back-end/index.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      where "gradeId" = $1
+    returning *;
+  `;
+
+  const params = [id];
+
+  db.query(sql, params)
+    .then(result => {
+      const grade = result.rows;
+      if (!grade) {
```
Comment (thebearingedge) [inline sgt-back-end/index.js]: Same as above, the array is truthy, but its length might be 0.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Excellent job so far, just a few changes.

File: sgt-back-end/index.js (hunk 1 above)
Comment (Lee-SungWon) [reply sgt-back-end/index.js]: revised!

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**